### PR TITLE
change log with changes from v3.0.15

### PR DESCRIPTION
**Pull request by bashbaug**
This change is ready for review, but I will keep it as a draft until all changes have been incorporated.

### CODE REVIEW

**Comment by bashbaug** [comment]
Need to update to new extension dictionaries after #1131 is merged.

**Review by alycm — APPROVED** [review]
LGTM (pending merging of https://github.com/KhronosGroup/OpenCL-Docs/pull/1103 of course).

**Pull request merged**
